docs_versioned_docs/version-ros2jazzy/components/yaml/sensors/wiferion.mdx

```diff
@@ -0,0 +1,52 @@
+<table>
+<tr>
+<!-- <td>
+<center>
+<figure>
+    <img src={require("./img/wiferion.png").default} width="250" />
```

Comment:
Is this image missing?

Reply:
Yes, I have no mesh for the charger receiver. I could add the current blue square.